Refresh AWS Discover doc page CTOR-1064 (#4641)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatch-discover.md

Reviewer: typo?

Reviewer: mandatory in FR

Reviewer: ```suggestion
> conditions that cannot be applied.
```

Reviewer: ```suggestion
> All the fields of this form are optional.
```

Author: ```suggestion
| elbv2:DdescribeLoadBalancers    |
```

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Connector dependencies

> The following monitoring connectors will be installed when you install the **Amazon API Gateway** connector through the

Reviewer: AWS Discover

> The following monitoring connectors will be installed when you install the **Amazon API Gateway** connector through the

Author: ```suggestion
The following monitoring connectors will be installed when you install the **AWS Discover** connector through the
```

| Amazon AWS EC2                  | Discover Amazon AWS EC2 instance             |
| Amazon AWS ASG                  | Discover Amazon AWS Auto Scaling Groups      |
| Amazon AWS EFS                  | Discover Amazon AWS EFS instances            |
| Amazon AWS FSX                  | Discover Amazon AWS FSX intances             |
| Amazon AWS Kinesis              | Discover Amazon AWS Kinesis streams hosts    |
| Amazon Web Service Lambda       | Discover AWS Lambda instances                |
| Amazon AWS RDS                  | Discover Amazon AWS RDS instances            |
| Amazon AWS S3                   | Discover Amazon AWS S3 instances             |
| Amazon Web Service SNS          | Discover AWS SNS Topics                      |
| Amazon Web Service SQS          | Discover AWS SQS queues                      |
| Amazon AWS VPN                  | Discover AWS VPN                             |

More information about discovering hosts automatically is available on the [dedicated page](/docs/monitoring/discovery/hosts-discovery).

### Collected metrics & status

No metrics or status for this monitoring connector.

## Prerequisites

### AWS Privileges

> Configure a service account (access/secret key combo)

Reviewer: ```suggestion
Configure a service account (access/secret key combo).
```

| sqs:ListQueues                 |
| cloudwatch:listMetrics         | 
| cloudwatch:getMetricStatistics |

### Plugin dependencies

To interact with the Amazon APIs, you can use either use the *awscli* binary provided by Amazon or *paws*, a Perl AWS SDK (recommended). You must install it on every poller expected to monitor AWS resources.

> > For now, it is not possible to use *paws* if you are using a proxy to reach the AWS Cloudwatch APIs.

Reviewer: Cloudwatch?
Not the same as FR

> > For now, it is not possible to use *paws* if you are using a proxy to reach the AWS Cloudwatch APIs.

Author: ```suggestion
> Please note that you cannot use paws if you are connecting via a proxy.
```

</TabItem>
</Tabs>

## Installing the monitoring connector

### Pack

 The installation procedures for monitoring connectors are slightly different depending on [whether your license is offline or online](../getting-started/how-to-guides/connectors-licenses.md).

1. If the platform uses an *online* license, you can skip the package installation
instruction below as it is not required to have the connector displayed within the
**Configuration > Connectors > Monitoring Connectors** menu.
If the platform uses an *offline* license, install the package on the **central server**
with the command corresponding to the operating system's package manager:

<Tabs groupId="sync">
<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">

```bash

> dnf install ccentreon-pack-cloud-aws\*

Reviewer: same as FR

apt install centreon-pack-cloud-aws\*
```

</TabItem>
<TabItem value="CentOS 7" label="CentOS 7">

```bash
yum install centreon-pack-cloud-aws\*
```

</TabItem>
</Tabs>

2. Whatever the license type (*online* or *offline*), install the **Amazon API Gateway** connector through
the **Configuration > Connectors > Monitoring Connectors** menu.

### Plugin

Since Centreon 22.04, you can benefit from the 'Automatic plugin installation' feature.
When this feature is enabled, you can skip the installation part below.

You still have to manually install the plugin on the poller(s) when:
- Automatic plugin installation is turned off
- You want to run a discovery job from a poller that doesn't monitor any resource of this kind yet

> More information in the [Installing the plugin](/docs/monitoring/pluginpacks/#installing-the-plugin) section.

Use the commands below according to your operating system's package manager:

<Tabs groupId="sync">
<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">

```bash
dnf install centreon-plugin-Cloud-Aws-Cloudwatch-Api
```

</TabItem>
<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">

```bash

> dnf install centreon-plugin-Cloud-Aws-Cloudwatch-Api

Reviewer: same as FR
